pavelib/quality.py

```diff
@@ -229,13 +229,29 @@ def run_complexity():
     For additional details on radon, see http://radon.readthedocs.org/
     """
     system_string = 'cms/ lms/ common/ openedx/'
-    print "--> Calculating cyclomatic complexity of files..."
+    complexity_report_dir = (Env.REPORT_DIR / "complexity")
+    complexity_report = complexity_report_dir / "python_complexity.log"
+
+    # Ensure directory structure is in place: metrics dir, and an empty complexity report dir.
+    Env.METRICS_DIR.makedirs_p()
+    _prepare_report_dir(complexity_report_dir)
+
+    print "--> Calculating cyclomatic complexity of python files..."
     try:
         sh(
-            "radon cc {system_string} --total-average".format(
-                system_string=system_string
+            "radon cc {system_string} --total-average > {complexity_report}".format(
+                system_string=system_string,
+                complexity_report=complexity_report
             )
         )
+        complexity_metric = _get_count_from_last_line(complexity_report, "python_complexity")
+        _write_metric(
+            complexity_metric,
+            (Env.METRICS_DIR / "python_complexity")
+        )
+        print "--> Python cyclomatic complexity report complete."
+        print "radon cyclomatic complexity score: {metric}".format(metric=str(complexity_metric))
```

Comment:
Should this wording be be in the file too too? I'm wondering if it will be more helpful to have there if we are going to  show `metrics/python_complexity` as a report on a build.